File: 1-easy/problems/1844-replace-all-digits-with-characters/solution.py
class Solution:
    def replaceDigits(self, s):
        """
        :type s: str
        :rtype: str
        """
        return "".join(chr(ord(s[i-1])+int(s[i])) if i%2 else s[i] for i in range(len(s)))

File: 1-easy/problems/1844-replace-all-digits-with-characters/solution.ts
function replaceDigits(s: any): boolean | number | string | any {
  """
  :type s: str
  :rtype: str
  """
  return "".join(chr(ord(s[i-1])+int(s[i])) if i%2 else s[i] for i in Array.from({length: (s}, (_, i) => i).length));
}

export { replaceDigits };
